src/gymFunctions.cpp

```diff
@@ -0,0 +1,169 @@
+#include "../include/kindyn/gymFunctions.h"
+#include <iostream>
+gymFunctions::gymFunctions(cardsflow::kindyn::Robot* robot, int id, bool respect_limits){
+    if (!ros::isInitialized()) {
+        int argc = 0;
+        char **argv = NULL;
+        ros::init(argc, argv, "Gym functions", ros::init_options::NoSigintHandler);
```

Comment:
What exactly are we initializing here? A node? Shall we name each node with a postfix, like the services?